### DISCUSSION TITLE
Launch Terminal At Current Location

### DISCUSSION THREAD

**Discussion by baiguai** [mucommander/mucommander#1105] [Ideas]
One feature I use ALL the time in Double Commander is the F9 hotkey, that launches a console window at the current panel's location. This would be SUPER cool in Mucommander, and would allow me to probably ditch other 'Commander' apps for MU!

Thank you!

**Comment by pskowronek** [comment]
@baiguai 
muC commander has got that for almost a year. Type F12, or CMD-R, or click `Run command` (you may associate your own shortcut). So I think you can ditch Double Commander ;) 

Pls let us know.

**Reply by baiguai** [reply]
Oh! Cool! Thank you. Yeah now I'll be rockin' Mucommander :)
